assets/models/organelles/Cilia.tscn

```diff
@@ -0,0 +1,368 @@
+[gd_scene load_steps=10 format=2]
+
+[ext_resource path="res://assets/models/organelles/Flagellum.glb" type="PackedScene" id=1]
+[ext_resource path="res://assets/textures/flagellum.png" type="Texture" id=2]
+[ext_resource path="res://shaders/Organelle.shader" type="Shader" id=3]
+[ext_resource path="res://assets/models/organelles/Flagellum.anim" type="Animation" id=4]
+[ext_resource path="res://assets/textures/dissolve_noise.tres" type="Texture" id=5]
+[ext_resource path="res://assets/models/organelles/Material_001.material" type="Material" id=6]
+
+[sub_resource type="ShaderMaterial" id=3]
+resource_local_to_scene = true
+shader = ExtResource( 3 )
+shader_param/dissolveValue = 0.0
+shader_param/outlineWidth = 0.05
+shader_param/growColor = Color( 0.521569, 0.521569, 0.521569, 1 )
+shader_param/tint = Color( 1, 1, 1, 1 )
+shader_param/texture = ExtResource( 2 )
+shader_param/dissolveTexture = ExtResource( 5 )
+
+[sub_resource type="ArrayMesh" id=1]
```

Comment:
This should be stored in separate file to make git lfs work (similarly to how the organelles have a .mesh files for them)